test/EFCore.MySql.IntegrationTests/Tests/Attributes.cs

```diff
@@ -1,11 +1,12 @@
-﻿using Xunit;
+﻿using Pomelo.EntityFrameworkCore.MySql.Storage.Internal;
+using Xunit;
 
 namespace Pomelo.EntityFrameworkCore.MySql.IntegrationTests.Tests
 {
-	public class SkipAppVeyorFact : FactAttribute {
+	public class SkipAppVeyorAndMariaDbFact : FactAttribute {
```

Comment:
Instead of tacking on `SkipAppVeyorAndMariaDbFact`, leave `SkipAppVeyorFact` and add a `SkipMariaDbFact`